stash_engine/app/views/stash_engine/dashboard/view_terms.html.erb

```diff
@@ -0,0 +1,17 @@
+<% @page_title = "Privacy Policy and Terms of Service" %>
+
+<h1>Privacy Policy</h1>
+
+<p>Dryad respects the privacy of our users and works to protect the personally identifiable information we collect,
+  which is limited to name and contact information. View our <a href="https://datadryad.org/pages/policies#privacy">full
+  privacy policy</a></p>
+
+<h1>Terms of Service</h1>
+
+<p>To use Dryad's data publishing services, you must agree to the <a href="https://datadryad.org/pages/policies">Terms
+  of Service</a></p>
+
+<div class="c-terms-of-service">
+  <%= button_to 'Yes. I accept the Terms of Service.', terms_accept_path %>
+  <%= button_to 'I disagree. Log me out.', sessions_destroy_path %>
```

Comment:
It looks like this does a POST request to the destroy path (what button_to defaults to) which the route for the destroy path isn't accepting, so it causes an error.  There are two possible solutions to this.

1) Add "method: \:get" to the end of the line and I believe it will fix it since it defaults to a POST with the button_to which the routes don't recognize.  See https://stackoverflow.com/questions/8684467/button-to-with-get-method-option-in-rails

or

2) Add a POST method as as acceptable for the sessions/destroy route, also in the routes file.

Honestly since this changes the state of something (the session) I suppose it makes some sense to use a POST request (or maybe even some other request like DELETE?) if we wanted to.

I think the GET got used because the link in the UI that goes directly to the session#destroy action, but . . . I could imagine putting in a bug report for it and changing the request types later if we want to.